src/PresentationalComponents/Export/SystemsPdfBuild.js

```diff
@@ -83,12 +84,12 @@ export const TablePage = ({ page, systems, intl }) => {
         .join(':')} UTC`;
       return [
         <Text key={system.system_uuid} style={styles.nameColumn}>
-          <Link
+          <InsightsLink
```

Comment:
Watch out! You are replacing React **PDF** `Link` component with a React Router `Link` (Wrapper component). This one can stay as it is.

Reply:
Yep, it works after I added Link back